### PR TITLE
Installing a new power infrastructure for Saltern.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Saltern still has the deprecated power network. We need to change that. I've deleted all traces of the old power system since our new power design is so different and needs a "clean slate" to both look good and be functional. The current mapping situation for SS14 makes me want to paint my walls red so here's a pr hoping someone else will do it for me.

- [x] Removed old power network from Saltern.
- [x] Deleted the deprecated power entities.
- [ ] Remapped the entire fucking station with the new power system.

[How to map](https://hackmd.io/@ss14/docs/%2F1CmC4RpnStSAIOKWWa1fyw) | [New power system](https://hackmd.io/@ss14/docs/%2FZmFcFhWIRGGVcnyzq9s1pw) | [Big Floppa](https://cdn.discordapp.com/attachments/277256016045932544/738974049123696720/0f59406ac56b82fab5fdd0fb16812dee7d0f9f52.mp4)

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
If someone wants to work on this PLEASE leave a comment. I'll update the map file and resolve conflicts so you have a fresh, up-to-date slate.

**Comment by juliangiebel** [comment]
I can do the mapping, i want to add emergency lights to the station anyway.
I would need #1830 to be merged first though.

**Comment by SweptWasTaken** [comment]
Looks like it's conflicting again. _screams_.

**Comment by SweptWasTaken** [comment]
Tests are failing because stationstation.yml which was fixed by #1963 . 

Julian is still going to commit his stripping of stationstation.yml so wait until that commit before merging. 

(This needs to be merged before any other map changes ftlog)

File: Resources/Prototypes/Entities/Constructible/Ground/wires.yml [hunk 1]
```diff
@@ -95,8 +95,13 @@
   parent: ApcExtensionCable
   components:
   - type: NodeContainer
-    nodes:
-    - !type:AdjacentNode
-      nodeGroupID: HVPower
-    - !type:AdjacentNode
-      nodeGroupID: Apc
+    nodeTypes: { HVPower : ["AdjacentNode"], Apc : ["AdjacentNode"] }
```
Comment (DrSmugleaf) [inline Resources/Prototypes/Entities/Constructible/Ground/wires.yml:98]: Is this supposed to change? It seems like a bad merge conflict from 01b10cb6876cbbd3e476cf97501f4a892a3e41f5, as "nodeTypes" is no longer read anywhere in NodeContainerComponent but "nodes" is.

File: Resources/Prototypes/Entities/Constructible/Ground/wires.yml [hunk 2]
```diff
@@ -95,8 +95,13 @@
...
+
+- type: entity
+  id: Generator
+  name: Depriciated Generator
+  parent: DebugGenerator
+  components:
+  - type: PowerSupplier
+    voltage: High
+    supplyRate: 100000
```
Comment (DrSmugleaf) [inline Resources/Prototypes/Entities/Constructible/Ground/wires.yml:107]: This should be removed as well, as all the other depriciated power prototypes are gone too.

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Constructible/Ground/wires.yml (hunk 2 above)
Comment (juliangiebel) [reply Resources/Prototypes/Entities/Constructible/Ground/wires.yml:107]: gaaah, it was automatically merged like that

File: Resources/Prototypes/Entities/Constructible/Ground/wires.yml (hunk 1 above)
Comment (juliangiebel) [reply Resources/Prototypes/Entities/Constructible/Ground/wires.yml:98]: Looks like this on latest master: 
`   nodes:
    - !type:AdjacentNode
      nodeGroupID: Apc`

**Comment by SweptWasTaken** [comment]
This is ready for a merge. Everyone thanks @juliangiebel for being an incredible human being and mapping all this in.

Please merge this ASAP so we can make mapping changes again. I will not allow this to get any conflicts.

**Review by DrSmugleaf — APPROVED** [review]

**Comment by SweptWasTaken** [comment]
Hallelujah

**Pull request merged**